internal/integration/api/selinux.go

```diff
@@ -336,7 +337,121 @@ func (suite *SELinuxSuite) TestSecurityState() {
 // TODO: test for system and CRI container labels
 // TODO: test labels for unconfined system extensions, pods
 // TODO: test for no avc denials in dmesg
-// TODO: start a pod and ensure access to restricted resources is denied
+
+// TestNoPtrace confirms ptracing system processes is prohibited in enforcing mode.
```

Comment:
I'm not sure, but perhaps we should create one pod and run all the assertions against it, so these tests run faster

Reply:
I think this one is fine